src/main/app/forms/validation/validationErrors.ts

```diff
@@ -7,4 +7,11 @@ export class ValidationErrors {
   static readonly VALID_OWED_AMOUNT_REQUIRED: string = 'Enter a valid amount owed'
   static readonly AMOUNT_REQUIRED: string = 'Enter an amount'
   static readonly AMOUNT_INVALID_DECIMALS: string = 'Enter valid amount, maximum two decimal places'
+  static readonly OPTION_REQUIRED: string = 'Choose option'
+  static readonly WHEN_WILL_YOU_PAY_OPTION_REQUIRED: string = 'Please select when you will pay'
+  static readonly FIRST_PAYMENT_AMOUNT_INVALID: string = 'Enter a valid amount of first payment'
+  static readonly INSTALMENTS_AMOUNT_INVALID: string = 'Enter a valid amount for equal instalments'
+  static readonly FUTURE_DATE: string = 'Enter a first payment date in the future'
+  static readonly INVALID_DATE: string = 'Enter a valid date of first payment'
+  static readonly SELECT_PAYMENT_SCHEDULE: string = 'Select how often they should pay'
```

Comment:
These all are specific to repayment plan and they overlap with what we have in `ccj/form/models/repaymentPlan`. Can you please create `export class RepaymentPlanValidationErrors` in this file and use it in the new page and CCJ page?

Reply:
ok